espnet2/gan_svs/espnet_model.py

```diff
@@ -37,6 +38,7 @@ class ESPnetGANSVSModel(AbsGANESPnetModel):
     @typechecked
     def __init__(
         self,
+        frontend: Optional[AbsFrontend],
```

Comment:
As we have discussed, can we change the name instead of Frontend? The current name would be confusing to other people.